x-pack/plugins/enterprise_search/server/lib/ml/start_ml_model_deployment.ts

```diff
@@ -43,7 +43,7 @@ export const startMlModelDeployment = async (
   // we're downloaded already, but not deployed yet - let's deploy it
   const startRequest: MlStartTrainedModelDeploymentRequest = {
     model_id: modelName,
-    wait_for: 'started',
+    wait_for: 'starting',
```

Comment:
This change has a side effect - the ELSER callout's "Start single-threaded" button will turn the status into "started"  while the model is starting. This is fine for two reasons:
- Starting the ELSER model takes about a second, so it is very unlikely the user will run a pipeline with the model in the starting phase;
- Currently the status label displays "started" both for the "starting" and "started" phase in the ELSER callout.